ols/src/llms/llm_loader.py

```diff
@@ -303,11 +303,11 @@ def _watson_llm_instance(self, api_url: str, api_key: str, project_id: str) -> L
             GenParams.MAX_NEW_TOKENS: self.llm_params.get("max_new_tokens", 512),
             GenParams.RANDOM_SEED: self.llm_params.get("random_seed", 42),
             GenParams.TEMPERATURE: self.llm_params.get("temperature", 0.05),
-            GenParams.TOP_K: self.llm_params.get("top_k", 10),
-            GenParams.TOP_P: self.llm_params.get("top_p", 0.95),
+            GenParams.TOP_K: self.llm_params.get("top_k", 50),
```

Comment:
This parameter might be too high for tasks that require precision. Since it is configurable I'm okay using this value as default.